GTBitsOfGood/canine-assistants
# User Invite Functionality

## User Invite Functionality

Issue Number(s): 122

What does this PR change and why?
- invite user button + modal
- error handling
- new backend POST to add a user (adds a user with name “Invited User” so the table column has an actual value)
- Status depends on value of acceptedInvite and isActive
- Also previously, changing the access level through the table refreshed the page, but now it does a table state update w/o refresh

### Checklist


### Critical Changes
for the user API schema (used for createUser), I changed
emailVerified: {
    type: Boolean,
    default: null,   >>> default: false,
  },

### How To Test

change values through MongoDB Compass to see the full changes (since the seed values have all “acceptedInvite” false); in the Users table, make some of the users “acceptedInvite” to true
